content/posts/networkx/GSoC-2022/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Hello World"
+date: 2022-06-09
+draft: false
+description: "Getting started"
+tags: ["gsoc", "networkx"]
+displayInList: true
+author: ["kpetridis24"]
+
+resources:
+  - name: featuredImage
+    src: "GSoC_logo.png"
+    params:
+      description: "Google Summer of Code Logo with NetworkX logo"
+      showOnTop: true
+---
+ 
+I got accepted as a GSoC contributor, and I am so excited to spend the summer working on 
+such an incredibly interesting project. The mentors are very welcoming, communicative, fun 
+to be around, and I really look forward to collaborate with them.
```

Comment:
This is a nice intro :) IMO it'd be good to include a sentence or two about what you'll be working on. You don't have to go into great detail or anything, but at least a mention of VF2++ and (if you're comfortable doing so) perhaps linking out to your GSoC proposal.